R/pandoc.R

```diff
@@ -497,7 +497,7 @@ find_pandoc <- function() {
 
     # determine the versions of the sources
     versions <- lapply(sources, function(src) {
-      if (file.exists(src))
+      if (dir_exists(src))
```

Comment:
In the case of #346, if `$HOME/opt/pandoc` is the pandoc executable instead of a dir, this path will be skipped, hence less confusing error message.